src/js/candidate-detail.js

```diff
@@ -0,0 +1,19 @@
+let urlParams = new URLSearchParams(window.location.search);
+let slug = urlParams.get('slug') || window.location.pathname.split('/')[2];
+// console.log(slug);
+// console.log(window.location.pathname);
+
+let html = document.getElementById('mustache-template').innerHTML;
+let htmlContainer = 'pageContent';
```

Comment:
Maybe we should add page specific sections to config.json where we could store stuff like this

Reply:
Sure, I don't see why not.